src/main/java/de/tum/in/www1/artemis/domain/Organization.java

```diff
@@ -0,0 +1,122 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import org.hibernate.annotations.BatchSize;
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@Entity
+@Table(name = "organization")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Organization extends DomainObject {
+
+    @NotNull
+    @Size(max = 100)
+    @Column(name = "name", length = 100)
+    private String name;
+
+    @NotNull
+    @Size(max = 50)
+    @Column(name = "shortName", length = 50)
+    private String shortName;
+
+    @Column(name = "url")
+    private String url;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "logoUrl")
+    private String logoUrl;
+
+    @NotNull
+    @Column(name = "emailPattern")
+    private String emailPattern;
+
+    @ManyToMany
+    @JoinTable(name = "user_organization", joinColumns = { @JoinColumn(name = "organization_id", referencedColumnName = "id") }, inverseJoinColumns = {
+            @JoinColumn(name = "user_id", referencedColumnName = "id") })
+    @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+    @BatchSize(size = 20)
```

Comment:
remove BatchSize and add JsonIgnoreProperties

Reply:
Done :+1: